pkg/controller/provider/web/vsphere/vm.go

```diff
@@ -69,6 +69,13 @@ func (h VMHandler) List(ctx *gin.Context) {
 	}
 	pb := PathBuilder{DB: db}
 	for _, m := range list {
+		if m.IsTemplate {
```

Comment:
This works only if the user requests the `model.MaxDetail` otherwise the database is not returning the template.
Could you please check how to add this to the list options?